tests/unit/api/BackbeatRequest.spec.js

```diff
@@ -0,0 +1,41 @@
+const assert = require('assert');
+
+const BackbeatRequest = require('../../../lib/api/BackbeatRequest');
+
+function setRouteDetails(bbRequest, route) {
+    bbRequest.setRoute(route);
+    bbRequest._parseRoute();
+    return bbRequest.getRouteDetails();
+}
+
+describe('BackbeatRequest helper class', () => {
+    let bbRequest;
+
+    before(() => {
+        bbRequest = new BackbeatRequest({ url: '' });
+    });
+
+    it('should not store any route details for healthcheck routes', () => {
+        const results = setRouteDetails(bbRequest, '/_/healthcheck');
+
+        assert.deepStrictEqual(results, {});
+    });
+
+    it('should parse routes and store internally as route details', () => {
+        const results = setRouteDetails(bbRequest, '/_/metrics/crr/all');
+        const routeDetails = bbRequest.getRouteDetails();
+
+        assert.strictEqual(results.category, routeDetails.category);
```

Comment:
I don't get the intent of these checks, since `results` comes from a call to `bbRequest.getRouteDetails()` already. It checks that results from `getRouteDetails` are consistent but that does not sound like very useful to check, did you intend some other test instead? If you expect some specific info in those fields you might check them directly against what you expect.